pom.xml

```diff
@@ -127,6 +128,19 @@
       <artifactId>structs</artifactId>
       <version>1.2</version>
     </dependency>
+
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>script-security</artifactId>
+      <version>1.37</version>
```

Comment:
I thought you were going to use https://github.com/jenkinsci/script-security-plugin/pull/169 here, so we can verify that the proposed API actually works for its supposed purpose? If so, you would need to `deploy` a snapshot of the upstream PR, then change this to a timestamped snapshot version, so CI can verify this PR.

Reply:
@jglick , I have verified the API works locally. Would you point me at how to deploy a timestamped snapshot?

Reply:
I think @jglick meant to put a timestamped snapshot dependency here. If you point me to the artifact you need I think I can make the change.

Reply:
Basically we need jenkinsci/script-security-plugin#169 built and deployed. Also we need 747a0bc6 reverted here so that the new API is used. Better let me first do little refactoring of script security pull request and then test it. I just have so much other work since new year. Hope to get back to this next week.